various-db-engines-it/db-it/oracle-it/pom.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.vertx.openshift</groupId>
+    <artifactId>db-it</artifactId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>oracle-it</artifactId>
+
+  <properties>
+    <vertx.verticle>io.vertx.openshift.oracledb.OracleVerticle</vertx.verticle>
+    <vertx.health.path>/healthcheck</vertx.health.path>
+    <nexus.repo>http://www.maven.example.com/</nexus.repo>
```

Comment:
Is it required?

Reply:
well I thought that if we want to automate these tests (i.e. using our xPaaS Jenkins instance), then we need to have a way to get the Oracle jdbc driver. One would be through this property, as we have an internal nexus repo containing Oracle jdbc driver, so I would just configure this property when running automated tests.